packages/lib/server/repository/profile.ts

```diff
@@ -375,9 +375,6 @@ export class ProfileRepository {
                 lockEventTypeCreationForUsers: true,
               },
             },
-            members: {
-              select: membershipSelect,
-            },
```

Comment:
This was causing to load more than 2k memberships for an enterprise customer in all pages